src/lpcjobqueue/schedd.py

```diff
@@ -5,10 +5,9 @@
 import re
 
 logger = logging.getLogger(__name__)
-os.environ["CONDOR_CONFIG"] = os.path.join("/srv/.condor_config")
+os.environ["CONDOR_CONFIG"] = os.path.join("/etc/condor/config.d/01_cmslpc_interactive")
```

Comment:
should we move this setting into the `.bashrc` generated by `bootstrap.sh`? that would further increase resiliency, i.e. even in cases of major changes, users would only have to rerun `bootstrap.sh` but could reuse their existing venv (rather than reinstalling `lpcjobqueue` entirely)